random-distribution-helpers.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+	"strings"
+	"time"
+)
+
+type RandomDistribution interface {
+	Sample() time.Duration
+	String() string
+}
+
+var _ RandomDistribution = RandomConstDistribution(nil)
```

Comment:
I think the right part does not compile... Instead of RandomConstDistribution(nil), use: RandomConstDistribution{}
Also for all other implementation types.